chargeOclock/Source/DataSources/ClientsDataSource.swift

```diff
@@ -27,10 +32,9 @@ class ClientsDataSource: NSObject, UITableViewDataSource {
 	}
 
 	func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-		let cell = tableView.dequeueReusableCell(withIdentifier: reuseIdentifier, for: indexPath)
-		if var clientCell = cell as? ClientSettable {
-			clientCell.client = clients[indexPath.row]
-		}
+		let cell = tableView.dequeueReusableCell(withIdentifier: reuseIdentifier, for: indexPath) as! ClientTableViewCell
```

Comment:
remove the forced cast, use the pattern that was used before, since it's much more robust.